include/openthread/ip6.h

```diff
@@ -415,19 +415,30 @@ typedef void (*otIp6ReceiveCallback)(otMessage *aMessage, void *aContext);
  */
 void otIp6SetReceiveCallback(otInstance *aInstance, otIp6ReceiveCallback aCallback, void *aCallbackContext);
 
+/**
+ * @struct otIp6AddressInfo
+ *
+ * This structure represents IPv6 address information.
+ *
+ */
+typedef struct otIp6AddressInfo
+{
+    const otIp6Address *mAddress;       ///< A pointer to the IPv6 address.
+    uint8_t             mPrefixLength;  ///< The prefix length of mAddress if it is a unicast address.
+    uint8_t             mScope : 4;     ///< The scope of this address.
+    bool                mIsAnycast : 1; ///< Whether this is an anycast address.
```

Comment:
```suggestion
    bool                mIsAloc : 1; ///< Whether this is an ALOC address.
```

Reply:
We care about whether it is a anycast or not, not whether it is a ALOC.